src/components/Root.tsx

```diff
@@ -1,35 +1,64 @@
-import React from 'react';
+import React, { FC } from 'react';
 import { I18nextProvider } from 'react-i18next';
+
+import { withContext, withToken } from '@graasp/apps-query-client';
+import { Loader, theme as graaspTheme } from '@graasp/ui';
+
 import { CssBaseline, ThemeProvider, createTheme, styled } from '@mui/material';
+import { grey, orange, pink } from '@mui/material/colors';
 import { StyledEngineProvider } from '@mui/material/styles';
-import pink from '@mui/material/colors/pink';
-import grey from '@mui/material/colors/grey';
-import orange from '@mui/material/colors/orange';
-import 'react-toastify/dist/ReactToastify.css';
-import { withContext, withToken } from '@graasp/apps-query-client';
-import { Loader } from '@graasp/ui';
-import i18nConfig from '../config/i18n';
-import App from './App';
 
+import i18nConfig from '../config/i18n';
+import {
+  CONTEXT_FETCHING_ERROR_MESSAGE,
+  TOKEN_REQUEST_ERROR_MESSAGE,
+} from '../config/messages';
 import {
-  queryClient,
   QueryClientProvider,
   ReactQueryDevtools,
   hooks,
+  queryClient,
 } from '../config/queryClient';
 import { showErrorToast } from '../utils/toasts';
+import App from './App';
+
+// declare the module to enable theme modification
+declare module '@mui/material/styles' {
+  interface Theme {
+    status: {
+      danger: { background: string; color: string };
+    };
+  }
+
+  // allow configuration using `createTheme`
+  interface ThemeOptions {
+    status?: {
+      danger?: { background: string; color: string };
+    };
+  }
+
+  interface PaletteOptions {
+    default: string;
+  }
+}
 
 const theme = createTheme({
+  ...graaspTheme,
   palette: {
-    primary: {
-      main: '#5050d2',
-    },
     secondary: pink,
     default: grey['500'],
     background: {
       paper: '#fff',
     },
   },
+  typography: {
+    h2: {
+      fontSize: '2rem',
+    },
+    h3: {
+      fontSize: '1.8rem',
+    },
```

Comment:
Just wondering, but if you need to change these why not use `h1` instead of h2?

Reply:
I think initially, I wanted to keep h1 for main titles in other views but I may change that if I don't use them this way.